loefsys/reservations/models/boat.py

```diff
@@ -3,8 +3,8 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
-from .choices import FleetChoices, ReservableCategories
-from .reservable import ReservableItem, ReservableType
+from .choices import FleetChoices
+from .reservable import ReservableItem
```

Comment:
Move multiline 1 deep relative imports to an `__init__` file, so you can use `from . import <imports here>`